### ISSUE TITLE
Action buttons hidden on Space rooms list on small window

### ISSUE THREAD

**Issue by luixxiul** [vector-im/element-web#21485]
### Steps to reproduce

1. Create a space
2. Add some rooms / spaces to the list
3. Make width of the window as small as possible

#### What did you expect?

The action buttons should be displayed on a small screen as they are critical and always should be available.

https://user-images.githubusercontent.com/3362943/159138363-168f3149-21e0-4d33-9380-7efbb15b68bf.mp4

#### What happened instead?

They are hidden on a small screen.

https://user-images.githubusercontent.com/3362943/159138372-84ee3245-2f5a-4a9d-9811-79d5b6bda24f.mp4

### Operating system

Debian

### Browser information

Firefox

### URL for webapp

localhost

### Application version

develop branch

### Will you send logs?

No
